internal/repository/master_index.go

```diff
@@ -261,6 +262,8 @@ func (mi *MasterIndex) MergeFinalIndexes() {
 	mi.idx = newIdx
 }
 
+const SaveIndexParallelism = 4
```

Comment:
That constant should start with a lower-case letter

Reply:
changed.